dali/pipeline/operators/fused/resize_crop_mirror.h

```diff
@@ -72,14 +72,33 @@ class ResizeCropMirrorAttr : protected CropAttr {
     if (resize_shorter_) {
       // resize_shorter set
       const int shorter_side_size = spec.GetArgument<float>("resize_shorter", ws, index);
+
+      bool max_size_enforced = spec.ArgumentDefined("max_size");
```

Comment:
I'd like this to be a vector of to have a separate argument for each dimension (max_width, max_height, max_depth?).

Reply:
Added in last commit